code/modules/heavy_vehicle/components/body.dm

```diff
@@ -118,4 +120,24 @@
 			air_supply = null
 		C.forceMove(src)
 		update_components()
-	else . = ..()
\ No newline at end of file
+	else . = ..()
+
+/obj/item/mech_component/chassis/repair_brute_damage(var/amt)
+	take_brute_damage(-(amt / CHASSIS_DAMAGE_MOD))
+
+/obj/item/mech_component/chassis/repair_burn_damage(var/amt)
+	take_burn_damage(-(amt / CHASSIS_DAMAGE_MOD))
```

Comment:
This is weird. You mean for this to heal 100 damage a pop? That seems excessive.

Reply:
this offsets the reduction the other proc imposes

Reply:
What reduction do you mean?

Reply:
take_brute_damage and take_burn_damage multiplies by the mod to determine its amount, so it's `amt / 4` atm. so by doing the reverse on the repair one, which calls the former procs, it cancels out, giving you the regular `amt`